src/codeu/chat/client/core/ConversationContext.java

```diff
@@ -87,9 +87,15 @@ private ConversationPayload getUpdated() {
     return payloads.hasNext() ? payloads.next() : null;
   }
 
+<<<<<<< HEAD
```

Comment:
These are merge conflict markers, and you should not commit code in an unmerged state like this. This is also bad because the code doesn't compile.  Please resolve these merge conflicts.